var/spack/repos/builtin/packages/masurca/package.py

```diff
@@ -12,10 +12,26 @@ class Masurca(Package):
        approaches."""
 
     homepage = "http://www.genome.umd.edu/masurca.html"
-    url      = "ftp://ftp.genome.umd.edu/pub/MaSuRCA/latest/MaSuRCA-3.2.3.tar.gz"
+    version('3.3.1', sha256='587d0ee2c6b9fbd3436ca2a9001e19f251b677757fe5e88e7f94a0664231e020',
```

Comment:
I think it would be sufficient to just add 3.3.1, and leave the prior versions as-is

Reply:
Remove those 3 extra files.

Can not download the data from the old ftp server for 3.2.3. ( ftp://ftp.genome.umd.edu/pub/MaSuRCA/latest) 

the earliest version in github is 3.2.4.